Python/samples/house-energy/sim/house_simulator.py

```diff
@@ -2,7 +2,21 @@
 import matplotlib.pyplot as plt
 
 class House():
-    def __init__(self, K: float=0.5, C: float=0.3, Qhvac: float=9, hvacON: float=0, occupancy: float=1, Tin_initial: float=30):
+    def __init__(self, 
+                K: float=0.5, 
+                C: float=0.3, 
+                Qhvac: float=9, 
+                hvacON: float=0, 
+                occupancy: float=1, 
+                Tin_initial: float=30, 
+                Tout_initial: float= 20,
+                Tout_amplitude: float=5,
+                Tset_temp_start: float = 25,
```

Comment:
so unfortunatley my current code can't handle a name like `Tset_temp_start` . it's looking for `key_param` and deciding what to do with it that way.